src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -103,11 +108,22 @@ protected SchemaTypes()
         /// </summary>
         /// <param name="schema">A schema for which this instance is created.</param>
         /// <param name="serviceProvider">A service provider used to resolve graph types.</param>
-#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
         public SchemaTypes(ISchema schema, IServiceProvider serviceProvider)
+            : this(schema, serviceProvider, (IEnumerable<IGraphTypeMappingResolver>?)serviceProvider.GetService(typeof(IEnumerable<IGraphTypeMappingResolver>)))
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance for the specified schema, and with the specified type resolver.
+        /// </summary>
+        /// <param name="schema">A schema for which this instance is created.</param>
+        /// <param name="serviceProvider">A service provider used to resolve graph types.</param>
+        /// <param name="graphTypeMappings">A service used to map CLR types to graph types.</param>
```

Comment:
A service ?

Reply:
Oops; before I updated it to use an enumerable.

Reply:
```suggestion
        /// <param name="graphTypeMappings">A list of <see cref="IGraphTypeMappingResolver"/> instances used to map CLR types to graph types.</param>
```

Reply:
Merged suggestion above.  Resolve if ok.